### DISCUSSION TITLE
【目录】关于在post页面中添加列表文章没有

### DISCUSSION THREAD

**Discussion by lzk821** [tangly1024/NotionNext#1978] [Q&A]
**描述bug**
在post页面中添加列表文章没有目录

**复现步骤**
在一个post页面添加一个列表用来整合文章，在文中中添加小标题后该页面不会出现目录

**期望的正常结果**
在文章中打开这个列表中的文章可以在侧栏中显示该文章的目录结构

**截图**
![bug77.png](https://s2.loli.net/2023/08/21/RBNsluKCnoGfkJj.png)
![bug.png](https://s2.loli.net/2023/08/21/EhUctBdeZfupSDY.png)

**环境**

- 操作系统: windows
- 浏览器 chrome
- NotionNext版本 4.0.13
- 主题 heo 其他也有此类情况

**补充说明**
如果将该文章单独写在一个新的post页面就不会出现此类问题

**Comment by tangly1024** [comment]
需要提供文章的url地址

**Comment by lzk821** [comment]
> 需要提供文章的url地址
url链接是：https://www.liash.link/7e03b88680104d2a837f84611db870c4
感谢大佬
